### PR TITLE
Create DEPLOYMENT.md

### PR DESCRIPTION
**Pull request by henrikstranneheim**
This PR adds/fixes:
- Deployment instructions for Hasta

**How to prepare for test**:
- Nothing to test

**Review:**
- [x] code approved by BS, KN
- [x] "Merge" approved by HS
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Review by karlnyr — COMMENTED** [review]
I think this looks correct. From what I know with information from @barrystokman  we do not yet have a environement to paxa at hasta. Either we add it into this PR as a block or we remove this information until the PR regarding the demux-stage paxa situation is settled?

**Comment by henrikstranneheim** [comment]
There is a PR here: https://github.com/Clinical-Genomics/Pax/pull/9

**Review by karlnyr — APPROVED** [review]
Well, as long as that is implemented I say that this should be added as well 👍

**Comment by henrikstranneheim** [comment]
Now you can paxa `demultiplexing-stage`
Screenshot 2021-02-17 at 12 18 07

**Comment by henrikstranneheim** [comment]
@barrystokman Anything you want to add?

File: DEPLOYMENT.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+When all tests done and successful and PR is approved, follow these steps:
+
+1. Bump version:
+    1. Merge branch to master
+    1. In your local git version of this repo
+        1. Checkout master by: `git checkout master`
+        1. Update local branch by: `git pull origin master`
+        1. Bumpversion according to specifications, eg. `bumpversion <patch/minor/major>`
+        1. Push commit directly to master `git push`
+        1. Push commit directly to master `git push --tag`
```
Comment (barrystokman) [inline DEPLOYMENT.md:12]:
```suggestion
        1. Push commit directly to master `git push --tags`
```

**Review by barrystokman — APPROVED** [review]
I spotted one typo, nothing else to add!

**Comment by henrikstranneheim** [comment]
@barrystokman Thx! Merging this now

**Pull request merged**